microcosm_pubsub/consumer.py

```diff
@@ -72,7 +73,11 @@ def parse_content(cls, consumer, message):
         Parse and validate the message.
 
         """
-        base_message = consumer.pubsub_message_codecs["_"].decode(message)
+        try:
```

Comment:
Is the idea, that these functions `parse_content` and `handle_message` should be factored out into something with a more easily overriden API so they can be overridden with logic which doesn't assume your messages have a `mediaType`.